Client/Ecosia/UI/PageAction/PageActionMenu.swift

```diff
@@ -208,11 +211,45 @@ extension PageActionMenu: NotificationThemeable {
 // MARK: Cell
 
 class PageActionMenuCell: UITableViewCell {
+    private weak var badge: UIView?
+    
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: .subtitle, reuseIdentifier: reuseIdentifier)
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    func isNew(_ isNew: Bool) {
+        if isNew {
+            if badge == nil {
+                let badge = UIView()
+                badge.translatesAutoresizingMaskIntoConstraints = false
+                badge.isUserInteractionEnabled = false
+                badge.backgroundColor = .theme.ecosia.primaryBrand
+                badge.layer.cornerRadius = 10
+                contentView.addSubview(badge)
```

Comment:
I wonder if the badge wouldn't collide with long labels if it's simply added to the contenVview. As we use a default table view cell with .subtitle style, I think we could set the badge as .accessoryView to prevent collision